### PR TITLE
WoE: implement Conscription

### PR DESCRIPTION
**Pull request by strib**
Issue: #2905

### CODE REVIEW

File: server/game/cards/06-WoE/Conscription.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                    !!context.player.opponent &&
+                    context.player.opponent.creaturesInPlay.length >
+                        context.player.creaturesInPlay.length,
+                trueGameAction: ability.actions.makeTokenCreature((context) => ({
+                    amount:
+                        context.player.opponent.creaturesInPlay.length -
+                        context.player.creaturesInPlay.length
+                }))
+            }),
+            effect: 'make {1} token creature{2}',
```
Comment (romanogit) [inline server/game/cards/06-WoE/Conscription.js:20]: I believe this is the default effect msg

Comment (strib) [reply server/game/cards/06-WoE/Conscription.js:20]: Unfortunately the `conditional` action replaces it with `do something conditionally` with no other info.  I didn't see a way to turn that off but let me know if there is.

Comment (romanogit) [reply server/game/cards/06-WoE/Conscription.js:20]: You can remove the condition and make `amount = 0` when the condition is not applicable
```
this.play({
                gameAction: ability.actions.makeTokenCreature((context) => ({
                    amount: !!context.player.opponent &&
                    context.player.opponent.creaturesInPlay.length >
                        context.player.creaturesInPlay.length ? 
                        context.player.opponent.creaturesInPlay.length -
                        context.player.creaturesInPlay.length : 0
                }))
            }),
```

Comment (strib) [reply server/game/cards/06-WoE/Conscription.js]: Fixed along with Election, thanks!

**Pull request merged**